requirements/requirements.md

```diff
@@ -14,27 +15,27 @@ A collection of requirements and scenarios, framing the scope of the Notary v2 p
 
 ## Goals
 
-Notary v2 aims to address the learnings and gaps of v1, while prioritizing a set of goals and [scenarios](./scenarios.md).
+Notary Project aims to address the learnings and limitations of [TUF-based implementation](https://github.com/notaryproject/notary), while establishing and prioritizing a set of goals and [scenarios](./scenarios.md) for new implementation (this repository).
 
 1. Offline signature creation
 1. Signatures attesting to authenticity and/or certification
 1. Maintain the original artifact digest and collection of associated tags, supporting existing dev through deployment workflows
 1. Multiple signatures per artifact, enabling the originating vendor signature, public registry certification and user/environment signatures
-1. Native persistance within an [OCI Artifact][oci-artifacts] enabled, [distribution-spec][distribution-spec] based registry
-1. Artifact and signature copying within and across [OCI Artifact][oci-artifacts] enabled, [distribution-spec][distribution-spec] based registries
+1. Native persistance within an [OCI Image][oci-image] enabled, [OCI distribution specification][oci-distribution] based registry
```

Comment:
From: https://github.com/notaryproject/notaryproject/pull/262#discussion_r1272600151
I think the changes are trying to deal with the detail for which manifest type is used, vs. what concept is being supported. OCI Artifacts, and with the pending changes to Image and Distribution specs, support any artifact type to be persisted in an OCI Distribution-based registry.  Changing OCI Artifact to OCI Image makes it sound like notation can only sign images, which isn't accurate.

Suggestion:
Native persistance within an 1.1 compliant [OCI distribution specification][oci-distribution] based registry